extensions-jvm/couchbase/integration-test/src/main/java/org/apache/camel/quarkus/component/couchbase/it/CouchbaseResource.java

```diff
@@ -18,34 +18,80 @@
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
+import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
 
-import org.apache.camel.CamelContext;
+import com.couchbase.client.java.kv.GetResult;
+import org.apache.camel.ConsumerTemplate;
+import org.apache.camel.ProducerTemplate;
+import org.apache.camel.component.couchbase.CouchbaseConstants;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
 import org.jboss.logging.Logger;
 
+import static org.apache.camel.component.couchbase.CouchbaseConstants.COUCHBASE_DELETE;
+import static org.apache.camel.component.couchbase.CouchbaseConstants.COUCHBASE_GET;
+
 @Path("/couchbase")
 @ApplicationScoped
+@Consumes(MediaType.TEXT_PLAIN)
 public class CouchbaseResource {
 
     private static final Logger LOG = Logger.getLogger(CouchbaseResource.class);
+    private static final long TIMEOUT = 100000;
 
-    private static final String COMPONENT_COUCHBASE = "couchbase";
     @Inject
-    CamelContext context;
+    ProducerTemplate producerTemplate;
+    @Inject
+    ConsumerTemplate consumerTemplate;
+
+    @ConfigProperty(name = "couchbase.connection.uri")
+    String connectionUri;
+    @ConfigProperty(name = "couchbase.bucket.name")
+    String bucketName;
+
+    @PUT
+    @Path("id/{id}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public boolean insert(@PathParam("id") String id, String msg) {
+        LOG.infof("inserting message %msg with id %id");
```

Comment:
Is this a logging feature %var_name ?

Reply:
thanks!! no it's a bug. I've corrected the logs, but then I revert the code. So thanks for pointing out this one